cypress/page/login.page.ts

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable require-jsdoc */
+class LoginPage {
+  private email: string;
+  private psw: string;
+  private signInBtn: string;
+
+  constructor() {
+    this.email = "#email";
+    this.psw = "#passwd";
+    this.signInBtn = "#SubmitLogin > span";
+  }
+
+  public login(): void {
+    cy.get(this.email).type("aperdomobo@gmail.com");
+    cy.get(this.psw).type("WorkshopProtractor");
+  }
+
+  public signIn(): void {
+    cy.get(this.signInBtn).click();
```

Comment:
You can add the click to the login button to the login method and remove this one